lib/screens/profile_screens/contribution/interactor/viewmodels/scores_view_model.dart

```diff
@@ -8,11 +8,22 @@ class ScoresViewModel extends Equatable {
   final ScoreModel? plantedScore;
   final ScoreModel? transactionScore;
 
-  const ScoresViewModel(
-      {this.contributionScore, this.communityScore, this.reputationScore, this.plantedScore, this.transactionScore});
+  const ScoresViewModel({
```

Comment:
moved these calls away from main profile page

Reply:
Is this ScoresViewModel used in other places if not you can removed.

I see the main change here was decoupling the Contribution screen from Profile screen.

Reply:
its used in the contribution screen. 

And yes, that is the main change, decoupling